libraries/chain/steem_evaluator.cpp

```diff
@@ -2432,13 +2384,20 @@ void claim_account_evaluator::do_apply( const claim_account_operation& o )
          });
       }
 
-      FC_ASSERT( uint64_t( current_subsidy ) <= gpo.available_account_subsidies, "There are not enough subsidized accounts to claim" );
+      FC_ASSERT( gpo.available_account_subsidies >= STEEM_ACCOUNT_SUBSIDY_PRECISION, "There are not enough subsidized accounts to claim" );
 
       _db.modify( gpo, [&]( dynamic_global_property_object& gpo )
       {
-         gpo.available_account_subsidies -= current_subsidy;
+         gpo.available_account_subsidies -= STEEM_ACCOUNT_SUBSIDY_PRECISION;
       });
    }
+   else
+   {
+      FC_ASSERT( o.fee == wso.median_props.account_creation_fee,
+         "Cannot pay more than account creation fee. paid: ${p} fee: ${f}",
```

Comment:
Can this error condition also be hit if the paid fee is less than the account creation fee?

Reply:
Yes,

```
if( o.fee.amount == 0 )
{
 ...
}
else
{
 FC_ASSERT( ... )
}
```

Reply:
And there is a unit test for the case of the op fee being between 0 and the account creation fee.